src/m365/teams/commands/meeting/meeting-get.ts

```diff
@@ -0,0 +1,135 @@
+import { Cli } from '../../../../cli/Cli';
+import Command from '../../../../Command';
+import { Logger } from '../../../../cli/Logger';
+import { AxiosRequestConfig } from 'axios';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from "../../../base/GraphCommand";
+import commands from '../../commands';
+import { Meeting } from '../Meeting';
+import { validation } from '../../../../utils/validation';
+import * as AadUserGetCommand from '../../../aad/commands/user/user-get';
+import { Options as AadUserGetCommandOptions } from '../../../aad/commands/user/user-get';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  userId?: string;
+  userName?: string;
+  email?: string;
+  joinUrl: string;
+}
+
+class TeamsMeetingGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MEETING_GET;
+  }
+
+  public get description(): string {
+    return 'Get specified meeting details';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        userId: typeof args.options.userId !== 'undefined',
+        userName: typeof args.options.userName !== 'undefined',
+        email: typeof args.options.email !== 'undefined'
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --userId [userId]'
+      },
+      {
+        option: '-n, --userName [userName]'
+      },
+      {
+        option: '--email [email]'
+      },
+      {
+        option: '-j, --joinUrl <joinUrl>'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (args.options.userId && !validation.isValidGuid(args.options.userId)) {
+          return `${args.options.userId} is not a valid Guid`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  #initOptionSets(): void {
+    this.optionSets.push(
+      ['userId', 'userName', 'email']
```

Comment:
We shouldn't use option sets here since we should be able to omit all 3 options to retrieve a meeting for the current user.